backend/src/main/kotlin/fr/gouv/cacem/monitorenv/infrastructure/api/adapters/bff/outputs/missions/MissionDataOutput.kt

```diff
@@ -26,6 +27,7 @@ data class MissionDataOutput(
     val createdAtUtc: ZonedDateTime? = null,
     val updatedAtUtc: ZonedDateTime? = null,
     val envActions: List<MissionEnvActionDataOutput>? = null,
+    val fishActions: List<MonitorFishMissionActionDataOutput>? = null,
```

Comment:
```suggestion
    val fishActions: List<MonitorFishMissionActionDataOutput> = listOf(),
```
Histoire d'avoir quelque chose de not-nullable dans le front ?

Reply:
ah oui petite erreur de ma part